Example/Podfile.lock

```diff
@@ -0,0 +1,22 @@
+PODS:
+  - FLEX (4.7.0)
```

Comment:
Is there any particular need to commit this file?

Let's keep this in gitignore if possible.

This way, we don't have to worry about updating this file once your changes are merged into `remove_private_api_usage`

Reply:
Done. reverted